.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
We do not want .vscode settings

Reply:
for code formatting to work, Editor settings are required. I can remove but this has no negative impact as VS code is one of the top most editor which is used. So having settings and growing these settings in future will help